### ISSUE TITLE
Uses sscanf for parsing files

### ISSUE THREAD

**Issue by turol** [WolfireGames/overgrowth#36]
`sscanf` is used for parsing files, for example
https://github.com/WolfireGames/overgrowth/blob/08da859107194d3d283c6b54565bffb656478a26/Source/Graphics/model.cpp#L422

This is 1. locale-dependent and 2. can be extremely slow, as seen recently with GTA5. Replace all of that with something better.

**Comment by feliwir** [comment]
I agree that we should try to change this. However to avoid any regression issues, i'd like to introduce unit tests for the file loading first. Afterwards i can work on changing the file loaders and keeping compability to previous behavior

**Comment by shinymerlyn** [comment]
@feliwir @autious do we have a unit testing system in tree yet? if not, any suggestions?

I have been exploring and documenting the file loader stuff enough that I'd be happy to write some unit tests, if we have a way to run them.
